### DISCUSSION TITLE
SimpleSDXL2-0802 release

### DISCUSSION THREAD

**Discussion by metercai** [metercai/SimpleSDXL#60] [Announcements]
2024.08.20 更新内容：
1，新架构进一步优化，提升在Windows环境的兼容性，压缩Fooocus和Comfy后端切换的资源消耗。
2，优化支持最低6G显存的Flux模型出图，提供质量优先的Fluxdev和速度优先的Flux+两个预置包，并可根据系统资源自动适配。
3，同步升级主线v2.5.5，优化增强修图UI，更符合Fooocus交互习惯。
 
升级方法：
1，模型下载及安装：
适合6G和8G显存的量化模型: https://hf-mirror.com/lllyasviel/flux1-dev-bnb-nf4/resolve/main/flux1-dev-bnb-nf4-v2.safetensors 和 https://hf-mirror.com/silveroxides/flux1-nf4-weights/resolve/main/flux1-schnell-bnb-nf4.safetensors 。下载后放到SimpleModels下的checkpoints目录下。
 
适合高配显卡的Flux全量模型和增补包: https://hf-mirror.com/metercai/SimpleSDXL2/resolve/main/flux1-dev.safetensors 和https://hf-mirror.com/metercai/SimpleSDXL2/resolve/main/flux1-schnell.safetensors ，下载后放到SimpleModels下的checkpoints目录下。Flux全量模型需要的增补包： https://hf-mirror.com/metercai/SimpleSDXL2/resolve/main/models_flux1_fp16_simpleai_patch.zip 。下载后放到SimpleAI目录下进行解压，覆盖到SimpleModels下，主要是补齐clip和vae目录下的模型。
 
2，程序包下载及安装：
新正式版程序包：https://hf-mirror.com/metercai/SimpleSDXL2/resolve/main/SimpleSDXL2_win.exe.7z0820 。下载后放到SimpleAI目录下，去掉.7z0820的文件名尾缀，双击运行exe文件，自解压到当前目录。新版本严禁与旧版本混装。
 
3，程序运行：
• 进入SimpleSDXL2_win_0820目录，双击启动脚本 run_SimpleSDXL.bat，完成第一次运行。
• 第一次运行后，在SimpleSDXL目录下会生成 config.txt文件，可以自行修改和添加模型路径。
• 模型根目录和出图目录可以自定义，但需用记事本打开启动脚本文件run_SimpleSDXL.bat，自行修改。
 
<img width="600" alt="目录结构" src="https://github.com/user-attachments/assets/9f1d9b2e-a753-494b-9883-210fc443346c">

**Comment by Yogesh-DevHub** [comment]
This is good, made a video on it. 
https://youtu.be/IREkiuRpQiY?si=953Qk6zgsOdCIRNd

**Reply by metercai** [reply]
> This is good, made a video on it. https://youtu.be/IREkiuRpQiY?si=953Qk6zgsOdCIRNd

thanks a lot, we will continue to update

**Comment by nitinh12** [comment]
how to use this in colab?

**Reply by metercai** [reply]
> colab

Because there are insurmountable walls in mainland China, the project has not been adapted for colab. You can refer to Fooocus for revisions.

**Reply by nitinh12** [reply]
> > colab
> 
> Because there are insurmountable walls in mainland China, the project has not been adapted for colab. You can refer to Fooocus for revisions.

Then how can we use it on runpod?
